app/root.tsx

```diff
@@ -133,12 +133,26 @@ function App() {
 	const theme = useTheme()
 
 	return (
-		<html lang="en" className={`${theme} h-full`}>
+		<html
+			lang="en"
+			className={`${theme} h-full`}
+			style={{ colorScheme: theme }}
```

Comment:
I'd prefer this to be set in the css file.

Reply:
Done! No need for javascript at all.

A very minor change in this PR but great that we don't have white scrollbars (in my chrome browser anyway) when in dark mode!